plugins/client/graphql-kotlin-client-generator/src/test/kotlin/com/expediagroup/graphql/plugin/client/generator/GenerateSharedResponseTypesIT.kt

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
in previous PR reviews it was mentioned that there is no need to have custom tests, we do have parametrized tests that helps to avoid having dedicated tests.